website/docs/components/form/select/partials/guidelines/guidelines.md

```diff
@@ -1,3 +1,11 @@
+!!! Info
+
+**This component is a native HTML element**
+
+Creating custom non-native variations of this component is not possible as we rely on the brower to render it as-is. More information can be found [on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select).
```

Comment:
Great addition! I found the original content to be easier to parse and more accurate (since the visual stylings aren't rendered "as-is"). Also updated the second sentence to use active voice instead of passive voice. 

```suggestion
**This component uses a native HTML element**

It's not possible to create custom variations of this component as it uses a native HTML element. Find more information [on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select).
```